### PR TITLE
Added jsp-api service

### PR DESCRIPTION
**Pull request by mroman-redhat**
Added parts of code necessary to run the JSP-API service necessary to run the new Spawner UI.

Also modified the interaction with the UI itself.

Related to: https://github.com/opendatahub-io/jupyterhub-singleuser-profiles/issues/41

### CODE REVIEW

File: .jupyter/jupyterhub_config.py [hunk 1]
```diff
@@ -257,4 +255,4 @@ def clean_environment(spawner):
 c.KubeSpawner.user_storage_class = os.environ.get("JUPYTERHUB_STORAGE_CLASS", c.KubeSpawner.user_storage_class)
 admin_users = os.environ.get('JUPYTERHUB_ADMIN_USERS')
 if admin_users:
-    c.Authenticator.admin_users = set(admin_users.split(','))
+    c.Authenticator.admin_users = set(admin_users.split(','))
```
Comment (vpavlin) [inline .jupyter/jupyterhub_config.py:258]: Missing new line

File: .s2i/bin/assemble [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+npm run build
+
+cd /opt/app-root/share/jupyterhub/static/
+mkdir jsp-ui
+cp -a /tmp/src/jsp/ui/build/. /opt/app-root/share/jupyterhub/static/jsp-ui
+
+cd /tmp/src/jsp/ui/templates/
+yes | cp -rf spawn.html page.html /opt/app-root/share/jupyterhub/templates/
+cd /tmp/src/jsp/ui/styles/
+yes | cp -rf style.less /opt/app-root/share/jupyterhub/static/less
```
Comment (vpavlin) [inline .s2i/bin/assemble:19]: new line

Comment (vpavlin) [inline .s2i/bin/assemble:19]: You should run fix-permissions at the end - e.g. https://github.com/thoth-station/s2i-minimal-notebook/blob/master/.s2i/bin/assemble#L128

**Comment by mroman-redhat** [comment]
Changing to another branch, closing this

**Pull request closed without merging**